pkg/plugin/windows/hnsstats/hnsstats_windows.go

```diff
@@ -96,34 +104,40 @@ func pullHnsStats(ctx context.Context, h *hnsstats) error {
 			if err != nil {
 				h.l.Error("Getting Vswitch ports failed", zap.Error(err))
 			}
+
 			for _, ep := range endpoints {
 				if len(ep.IpConfigurations) < 1 {
-					h.l.Info("Skipping endpoint without IPAddress", zap.String("EndpointID", ep.Id))
+					h.l.Info("Skipping endpoint without IPAddress", zap.String(zapEndpointIDField, ep.Id))
 					continue
 				}
-				var mac string = ep.MacAddress
-				var ip string = ep.IpConfigurations[0].IpAddress
-				// var id string = ep.Id
-				if stats, err := hcsshim.GetHNSEndpointStats(ep.Id); err != nil {
-					h.l.Error("Getting endpoint stats failed for endpoint ID "+ep.Id, zap.Error(err))
+
+				id := ep.Id
+				mac := ep.MacAddress
+				ip := ep.IpConfigurations[0].IpAddress
+
+				if stats, err := hcsshim.GetHNSEndpointStats(id); err != nil {
+					h.l.Error("Getting endpoint stats failed", zap.String(zapEndpointIDField, id), zap.Error(err))
 				} else {
 					hnsStatsData := &HnsStatsData{hnscounters: stats, IPAddress: ip}
-					// h.l.Info(fmt.Sprintf("Fetched HNS endpoints stats for ID: %s, IP %s, MAC %s", id, ip, mac))
+					h.l.Info("Fetched HNS endpoints stats", zap.String(zapEndpointIDField, id),
```

Comment:
should we change this to Debug? printing this for every endpoint on the interval could be pretty noisy, but I don't have any output in front of me

Reply:
Updated, thanks for spotting that, Mathew!